### DISCUSSION TITLE
Starting a second server but same machine?

### DISCUSSION THREAD

**Discussion by TangJames** [thijsvanloef/palworld-server-docker#49] [Q&A] [answered]
Do you know how I would start a second server in another container but on the same machine? I know how to compose it and set it up in another container but how would the networking work with the public IP address?

**Comment by larrydemons** [comment]
set different port

**Comment by xHyperElectric** [comment, marked as answer]
Both your query port and your game port would have to be different. By default they are 27015 and 8211, so your second server you could set them as 27016 and 8212. You would have to port forward those two additional ports as well.
